sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -785,14 +785,15 @@ public Bound() {
             WriteDisposition.WRITE_EMPTY, true, null);
       }
 
-      private Bound(String name, String jsonTableRef,
-          SerializableFunction<BoundedWindow, TableReference> tableRefFunction, TableSchema schema,
+      private Bound(String name, @Nullable String jsonTableRef,
+          @Nullable SerializableFunction<BoundedWindow, TableReference> tableRefFunction,
+          @Nullable String jsonSchema,
           CreateDisposition createDisposition, WriteDisposition writeDisposition,
           boolean validate, BigQueryServices testBigQueryServices) {
         super(name);
         this.jsonTableRef = jsonTableRef;
         this.tableRefFunction = tableRefFunction;
-        this.schema = schema;
+        this.jsonSchema = jsonSchema;
         this.createDisposition = createDisposition;
         this.writeDisposition = writeDisposition;
         this.validate = validate;
```

Comment:
That the tests now fail is a symptom, I think, that we did not set the defaults in the initial constructor. But feel free to revert if it's not easy to fix, as this is only a minor user win.